packages/core/src/button/useButton.ts

```diff
@@ -83,10 +86,11 @@ export const useButton = <T extends Element>({
 
   const buttonProps = {
     "aria-disabled": disabled && focusableWhenDisabled ? true : undefined,
+    "data-loading": loading,
```

Comment:
Is this needed?

Reply:
Well, it doesn't hurt. I mean it is a piece of information that gets passed to the react component, so it makes sense to reflect on the DOM too. React-Aria and Mantine do pass it down, whereas shadcn and radix don't.

- https://mantine.dev/core/button/#loading-state
- https://react-spectrum.adobe.com/react-aria/Button.html#pending